### PR TITLE
Fix missing 2.5.5-SNAPSHOT changes

### PR DESCRIPTION
**Pull request by cweitkamp**
- Fix missing 2.5.5-SNAPSHOT changes

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: itests/org.openhab.binding.astro.tests/itest.bndrun [hunk 1]
```diff
@@ -45,5 +45,6 @@ Fragment-Host: org.openhab.binding.astro
 	org.eclipse.jetty.servlet;version='[9.4.20,9.4.21)',\
 	org.eclipse.jetty.util;version='[9.4.20,9.4.21)',\
 	org.openhab.core.test;version='[2.5.0,2.5.1)',\
-	org.openhab.binding.astro;version='[2.5.4,2.5.5)',\
-	org.openhab.binding.astro.tests;version='[2.5.4,2.5.5)'
+	jaxb-api;version='[2.2.11,2.2.12)',\
```
Comment (cpmeister) [inline itests/org.openhab.binding.astro.tests/itest.bndrun:48]: We are adding jaxb api now?

File: itests/org.openhab.binding.wemo.tests/itest.bndrun [hunk 2]
```diff
@@ -67,6 +67,7 @@ Fragment-Host: org.openhab.binding.wemo
 	org.eclipse.jetty.websocket.common;version='[9.4.20,9.4.21)',\
 	org.eclipse.jetty.xml;version='[9.4.20,9.4.21)',\
 	org.ops4j.pax.web.pax-web-api;version='[7.2.11,7.2.12)',\
-	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
```
Comment (cpmeister) [inline itests/org.openhab.binding.wemo.tests/itest.bndrun:70]: You may have removed logback classic, but logback core is still imported at the top.

File: itests/org.openhab.io.hueemulation.tests/itest.bndrun [hunk 3]
```diff
@@ -75,6 +75,7 @@ Fragment-Host: org.openhab.io.hueemulation
 	org.ops4j.pax.web.pax-web-jetty;version='[7.2.11,7.2.12)',\
 	org.ops4j.pax.web.pax-web-runtime;version='[7.2.11,7.2.12)',\
 	org.ops4j.pax.web.pax-web-spi;version='[7.2.11,7.2.12)',\
-	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
```
Comment (cpmeister) [inline itests/org.openhab.io.hueemulation.tests/itest.bndrun:78]: same here

File: itests/org.openhab.binding.tradfri.tests/itest.bndrun [hunk 4]
```diff
@@ -57,6 +57,7 @@ Fragment-Host: org.openhab.binding.tradfri
 	org.eclipse.californium.core;version='[2.0.0,2.0.1)',\
 	org.eclipse.californium.element-connector;version='[2.0.0,2.0.1)',\
 	org.eclipse.californium.scandium;version='[2.0.0,2.0.1)',\
-	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
```
Comment (cpmeister) [inline itests/org.openhab.binding.tradfri.tests/itest.bndrun:60]: here too

File: itests/org.openhab.binding.nest.tests/itest.bndrun [hunk 5]
```diff
@@ -65,6 +65,7 @@ Fragment-Host: org.openhab.binding.nest
 	org.eclipse.jetty.websocket.client;version='[9.4.20,9.4.21)',\
 	org.eclipse.jetty.websocket.common;version='[9.4.20,9.4.21)',\
 	org.eclipse.jetty.xml;version='[9.4.20,9.4.21)',\
-	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
```
Comment (cpmeister) [inline itests/org.openhab.binding.nest.tests/itest.bndrun:68]: logback is still used in the modbus tests, so I don't know if you forgot to switch it or had another reason to leave it...

**Comment by kaikreuzer** [comment]
@cpmeister I assume that @cweitkamp did not manually do any of those changes, but the bnd resolver did it. I thus wouldn't worry too much about the changes as long as it helps getting the build working ;-)

**Comment by cpmeister** [comment]
@kaikreuzer fair enough. Just wanted to check though. But I do agree that none of these changes are breaking.

**Comment by cweitkamp** [comment]
Yes, I just run `-DwithResolver` but you are right about the `jaxb-api`. It should not be included. We should put it onto the runblacklist.

**Comment by cpmeister** [comment]
I'm starting to think this error won't go away without some kind of intervention:
```
[ERROR] org.codehaus.plexus.archiver.ArchiverException: Error while expanding /home/travis/.m2/repository/org/openhab/osgiify/de.kaizencode.tchaikovsky/0.0.1/de.kaizencode.tchaikovsky-0.0.1.jar: archive is not a ZIP archive
```

Ideas?

**Comment by J-N-K** [comment]
I think it‘s caching. I‘m on the phone right now, so now way to check, but could you have a look at our repo if the jar there is valid? If it is, we‘ll probably have to wait until the maven cache expires.

File: itests/org.openhab.binding.astro.tests/itest.bndrun (hunk 1 above)
Comment (cweitkamp) [reply itests/org.openhab.binding.astro.tests/itest.bndrun]: Removed.

**Pull request merged**